rights-statements.ttl

```diff
@@ -28,202 +30,219 @@
 # 2015-07-17: updated from document to use correct URI patterns
 # 2015-08-11: update to make individual statements skos:Concepts, and to 
 #             use skos:note to model the Notices
+# 2015-09-16: updated text to match detailed Rights Statements document
+# 2015-09-16: removed (commented) OOC-NC. It is undecided whether this statement
+#             will be included in the final set. If it is re-added, the text should
+#             be revised.
 
-<InC> a dcterms:RightsStatement, skos:Concept ;
+<InC/1.0/> a dcterms:RightsStatement, skos:Concept ;
   skos:prefLabel "In Copyright"@en ;
-  dcterms:modified "2015-07-17" ;
+  dcterms:modified "2015-09-16" ;
   skos:closeMatch <http://www.europeana.eu/rights/rr-f/> ;
-  skos:definition """This item is protected by copyright and/or related rights.
-
-You are free to use this item in any way that is permitted by the copyright and related rights legislation that applies to you.
-
-For other uses you need to obtain permission from the rightsholders."""@en ;
-  skos:note "Unless expressly stated otherwise, the organisation that has made this item available makes no warranties about the item. You are responsible for your own use."@en ;
-  skos:note "You may find additional information about the copyright status of the item on the website of the organisation that has made the item available."@en ;
-  skos:scopeNote "This rights statement can be used for any work that is in copyright. Using this statement implies that the the organisation making this item available has determined that the item is in copyright and; is either the copyright holder, has obtained permission to make available the work from the copyright holder or, makes the work available under an exception or limitation to copyright (including fair use) that entitles it to make the work available."@en ;
+  skos:definition """This item is protected by copyright and/or related rights. 
+
+You are free to use this item in any way that is permitted by the copyright and related rights legislation that applies to your use. 
+
+For other uses you need to obtain permission from the rights-holder(s)."""@en ;
+  skos:note "Unless expressly stated otherwise, the organization that has made this item available makes no warranties about the item and cannot guarantee the accuracy of this rights statement. You are responsible for your own use."@en ;
+  skos:note "You may need to obtain other permissions for your intended use. For example, other rights such as publicity, privacy or moral rights may limit how you may use the material."@en ;
+  skos:note "You may find additional information about the copyright status of the item on the website of the organization that has made the item available."@en ;
+  skos:scopeNote """This rights statement can be used for an item that is in copyright. Using this statement implies that the organization making this item available has determined that the item is in copyright and either: 
```

Comment:
I'm glad you raised that. The formatting here is cumbersome. It would be nice to not have to interpret this as anything other than plain text.

Can we move forward with this, in the meanwhile, and make adjustments for the outcome of that discussion in a later PR?